content/docs/command-reference/exp/run.md

```diff
@@ -118,6 +118,12 @@ changes between/after queueing runs.
 `dvc exp run --temp`, for example to continue working on the project meanwhile
 (e.g. on another terminal).
 
+> ⚠️ Note that only tracked code and data will be included in `--queue/temp`
+> experiments. To include untracked files or directories, stage them with
+> `git add (-N)` first (before `dvc exp run`). Feel free to `git reset` them
+> afterwards. Git-ignored files/dirs are explicitly excluded from runs outside
+> the workspace.
```

Comment:
```suggestion
> the workspace to avoid tracking unnecessary or sensitive information.
```
Wording might not be right, but might be good to include rationale.

Reply:
This comment is new.

Reply:
I'll add this info.